src/de/grobox/liberario/TransportNetworks.java

```diff
@@ -579,4 +610,12 @@ TransportNetwork getTransportNetwork(String idString) {
 	private String getString(int res) {
 		return context.getString(res);
 	}
+
+	private String region(String name, String flag) {
+		return flag + " " + name;
+	}
+
+	private String description(String desc, String networks) {
```

Comment:
I thought a bit more about this:

- This is only helpful for users of Android-Studio (or editors that do that visual replacement)
- When in use, it is a nice feature. But normally the strings are always formed the same and the benefit is not very large.
- For those where the editor does not do this, this is just more boilerplate.

So I think I would prefer now the option that you proposed. I have added a commit to this PR.